java-sdk/connectors/feed-connector/README.md

```diff
@@ -65,3 +65,9 @@ Note: If after some subsequent crawls, all the documents are removed:
  - clear the datasource (checkpoint wil be removed)
  - change some properties if desired and save the datasource. 
  - Start crawling again
+
+## Upgrading a feed based connector to SDK version 2.0.1 
+
+The SDK version 2.0.1 brings changes to the tagging and removing of stray items. These are areas to update when upgrading your feed connector to SDK 2.1.0 or above.
```

Comment:
It should be 2.0.1 instead of 2.1.0 right?

Reply:
Hi @puneetkhanal  thanks. My mistake. Ill change this